### PR TITLE
fix(contacts): contact not shown on table

### PR DESCRIPTION
**Pull request by radireddy**
when a contact is added, its not shown on the table. A manual refresh is required to see it. Same issue with delete contact

Closes #MBP-535

### CODE REVIEW

File: packages/manager/modules/pci/src/projects/project/contacts/add/add.html [hunk 1]
```diff
@@ -1,20 +1,21 @@
...
+        data-title="{{ ::'cpb_rights_add_title' | translate }}"
+        data-primary-action="$ctrl.add()"
+        data-primary-label="{{ ::'cpb_rights_common_add' | translate }}"
+        data-secondary-action="$ctrl.goToContactsPage()"
+        data-secondary-label="{{ ::'cpb_rights_common_cancel' | translate }}"
+        data-on-dismiss="$ctrl.goToContactsPage()"
+        data-loading="$ctrl.isAdding">
 
-        <oui-field data-label="{{ :: $ctrl.model.isUSorCA ? 'cpb_rights_table_email' : 'cpb_rights_table_nichandle' | translate }}">
+        <oui-field data-label="{{ :: $ctrl.user.ovhSubsidiary === 'FR' ? 'cpb_rights_table_nichandle' : 'cpb_rights_table_email' | translate }}">
```
Comment (frenautvh) [inline packages/manager/modules/pci/src/projects/project/contacts/add/add.html]: warning i think you modify the behavior here, original test was : `indexOf(['US', 'CA'], me.country.toUpperCase())`

is it intentional ?

Comment (radireddy) [reply packages/manager/modules/pci/src/projects/project/contacts/add/add.html]: yes, both are same, if not US and CA , its FR. Let me know if I am wrong.

Comment (radireddy) [reply packages/manager/modules/pci/src/projects/project/contacts/add/add.html]: I think I am wrong here, let me change. Thanks for pointing this big mistake :)

File: packages/manager/modules/pci/src/projects/project/contacts/contacts.controller.js [hunk 2]
```diff
@@ -1,157 +1,59 @@
...
-    self.getRights();
-    initContact();
-  };
+  $onInit() {
+    this.model = {
+      owner: null,
+      billing: null,
+      isAdmin: false,
+    };
+    this.model.owner = this.projectInfo.contactAdmin;
```
Comment (marie-j) [inline packages/manager/modules/pci/src/projects/project/contacts/contacts.controller.js]: You could directly fill the model 
```suggestion
    this.model= {
     owner: this.projectInfo.contactAdmin,
     billing: this.projectInfo.contactBilling,
     isAdmin: this.user.nichandle === this.projectInfo.contactAdmin
     }
```

**Comment by jleveugle** [comment]
Hello @radireddy,
Your pull request is now closed due to inactivity.
Feel free to re-open it if you have any update.
Anna Bot for OVH UX <3

**Pull request closed without merging**